Header header
string[] ssids
string[] macs
uint16[] channels
int16[] strengths
int32[] ms
bool[] changed_signal_strength
